Gemfile

```diff
@@ -2,5 +2,8 @@ source "https://rubygems.org"
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-# Specify your gem's dependencies in datarockets-style.gemspec
 gemspec
+
+gem "pry-byebug"
```

Comment:
Why is rake `moved` and other 2 gems added here? Aren't they working being specified in the gemspec?

Reply:
no, these are just development dependencies. We run bundle only for development and I decided not to add them to gemspec.

Reply:
Well, that's exactly what `add_development_dependency` is doing. Any reasons not to use it?

Reply:
I don't think so. I removed it from gemspec. I checked some gems and some of them use that approach and works well. And I like the approach with adding development dependencies to Gemfile more

Reply:
Hm, even Bundler's author is recommending to put everything into the gemspec, except of some special cases - https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

Also, there was an ardent discussion on RubyGems, where the maintainers were not convinced to deprecate the `add_development_dependency` method, stating that the DRY way is to keep all the dependencies in one place - https://github.com/rubygems/rubygems/issues/1104

Otherwise, it is certainly up to you, as you're the gem author :)

Reply:
@texpert cool, thank you. I'll create a separate issue for that and check it later one more time.

Just quick thoughts: I still think that we can save dev dependencies in Gemfile, but I'm agreed with a point about `Gemfile.lock`

Reply:
https://github.com/datarockets/datarockets-style/issues/124